package.json

```diff
@@ -77,7 +83,10 @@
     "@types/tar": "6.1.10",
     "@typescript-eslint/eslint-plugin": "5.18.0",
     "@typescript-eslint/parser": "5.18.0",
+    "aws-cdk-lib": "^2.110.1",
     "aws-sdk-client-mock": "^3.0.0",
+    "constructs": "^10.3.0",
```

Comment:
Unused package?

Reply:
even though is not imported, it is needed to compile the CDK template